db/sql/400_api.sql

```diff
@@ -91,4 +93,16 @@ RETURNS setof json as $$
   SELECT cast(ST_AsGeoJSON(sc.*) AS json)
   FROM selected_cols AS sc;
 $$ LANGUAGE SQL VOLATILE;
+COMMENT ON FUNCTION api.get_observations_with_null_stop_id_4326 IS
+'Returns observations with null stop_id as GeoJSON features';
 
+CREATE OR REPLACE FUNCTION api.get_percentile_circles_with_stop_id(stop_id int)
+RETURNS setof json as $$
+  WITH selected_cols AS (
+    SELECT * FROM stopcorr.view_percentile_circles pc WHERE pc.stop_id = $1
+  )
+  SELECT cast(ST_AsGeoJSON(sc.*) AS json)
+  FROM selected_cols AS sc;
+$$ LANGUAGE SQL VOLATILE;
```

Comment:
Same here as above, read-only function that can be `STABLE` instead of `VOLATILE`.